cloud/deployment/deployment.go

```diff
@@ -7,18 +7,21 @@ import (
 	"sort"
 	"strconv"
 
+	airflowversions "github.com/astronomer/astro-cli/airflow_versions"
 	astro "github.com/astronomer/astro-cli/astro-client"
 	"github.com/astronomer/astro-cli/config"
 	"github.com/astronomer/astro-cli/pkg/ansi"
+	"github.com/astronomer/astro-cli/pkg/httputil"
 	"github.com/astronomer/astro-cli/pkg/input"
 	"github.com/astronomer/astro-cli/pkg/printutil"
 	"github.com/astronomer/astro-cli/pkg/util"
 	"github.com/pkg/errors"
 )
 
 var (
-	errInvalidDeploymentKey = errors.New("invalid deployment selection")
-	errNoDeployments        = errors.New("no Deployments found in this Workspace")
+	errInvalidDeployment    = errors.New("the Deployment specified was not found in this workspace. Your account or API Key may not have access to the deployment specified")
+	errInvalidDeploymentKey = errors.New("invaled Deployment selected")
+	noDeployments           = "no Deployments found in this Workspace, would you like to create one now?"
```

Comment:
Change to "No Deployments found in this Workspace. Would you like to create one now?"

Reply:
okay